src/tests/JNIGridnetClientSelfPlay.java

```diff
@@ -124,6 +157,7 @@ public byte[] render(boolean returnPixels) throws Exception {
     }
 
     public void gameStep(int[][] action1, int[][] action2) throws Exception {
+    	// FIXME do we really need to create this TraceEntry object at all?
```

Comment:
I don't think this is needed either. A "TraceEntry" should be only created if it is later added to a "Trace". A "Trace" is used to save/load replays of games. But I do not see that happening in this file. So, this is unnecessary. See the "tests/trace/TraceGenerationTest.java" file for an example. I bet that one example file that created a trace was used as a template to create this java file, and this is a leftover from that.